### PR TITLE
gh 0.10.0

### PR DESCRIPTION
**Pull request by mislav**
Created by https://github.com/mislav/bump-homebrew-formula-action

### CODE REVIEW

File: Formula/gh.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Gh < Formula
   desc "GitHub command-line tool"
   homepage "https://github.com/cli/cli"
-  url "https://github.com/cli/cli/archive/v0.9.0.tar.gz"
-  sha256 "318295e5a662f785662751f1e2cd4b1f613ec3aced1c4e7f1755d27922dbfdbf"
+  url "https://github.com/cli/cli/archive/v0.10.0.tar.gz"
+  sha256 "5d7474a89cd0d284387f1b58dc8b1771ebae8fc83cfa2410b354435b295c07df"
   revision 1
```
Comment (miccal) [inline Formula/gh.rb:6]: Please remove the `revision 1` line, thank you.

**Comment by mislav** [comment]
@miccal This is done and the `test` logic is updated for functionality changes; thank you!

**Comment by dawidd6** [comment]
Squash your commits please.

**Comment by mislav** [comment]
> Squash your commits please.

@dawidd6 Sure, but how does that help you, exactly? To squash commits on my end, I need to go through multiple steps of checkout, rebase, and force-pushing the branch. To squash the commit on your end as maintainer, you only need to click Squash and merge in the web UI. 

Did I miss any step of [the CONTRIBUTING doc](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?

**Comment by issyl0** [comment]
We [don't use the GitHub merge buttons to merge PRs](https://docs.brew.sh/Brew-Test-Bot-For-Core-Contributors) (link for info, not expecting you to have read it previously!), so either us or you will have to `gh pr checkout 56176` and do the rebase and force-push to your branch else we'll have extra commits that clutter the history needlessly.

**Comment by issyl0** [comment]
According to those docs, we could do this ourselves at merge-time with `brew pr-pull`, but for _reasons_ that I have yet to dig into it doesn't work with GPG signed commits, which is a setting that a lot of maintainers have turned on by default. 🌱

**Comment by mislav** [comment]
@issyl0 Thank you for explaining! I've squashed the commits

> we could do this ourselves at merge-time with `brew pr-pull`, but for _reasons_ that I have yet to dig into it doesn't work with GPG signed commits

Not even after this https://github.com/Homebrew/brew/pull/7243?

I noticed that Homebrew maintainers need to ask someone like me to squash their commits multiple times a week. Would it be useful to you if I contributed an Action that fails the build for community PRs that have more than one commit?

Or at least to add this rule to CONTRIBUTING?

**Comment by issyl0** [comment]
> Thank you for explaining! I've squashed the commits.

❤️

> > we could do this ourselves at merge-time with `brew pr-pull`, but for _reasons_ that I have yet to dig into it doesn't work with GPG signed commits
> 
> Not even after this [Homebrew/brew#7243](https://github.com/Homebrew/brew/pull/7243)?

Unfortunately not, it pulls the commits from the PR but then gives me a bad `git status` with the bottle block changes uncommitted. :shrug:

> I noticed that Homebrew maintainers need to ask someone like me to squash their commits multiple times a week. Would it be useful to you if I contributed an Action that fails the build for contributor PRs that have more than one commit?
>
> Or at least to add this rule to CONTRIBUTING?

Adding it to `CONTRIBUTING.md` or even the PR template (some will disagree with me on this as it's already quite long!) would be good. But the real fix would be to fix `brew pr-pull` to work properly, which is on us.

**Pull request closed without merging**